cinema/views.py

```diff
@@ -43,6 +60,25 @@
 
         return MovieSerializer
 
+    def get_queryset(self):
+        queryset = Movie.objects.all()
+        genres = self.request.query_params.get("genres")
+        actors = self.request.query_params.get("actors")
+        title = self.request.query_params.get("title")
+        if genres:
+            queryset = queryset.filter(
+                genres__in=split_int_string_by_comma(genres)
+            )
+        elif actors:
+            queryset = queryset.filter(
+                actors__in=split_int_string_by_comma(actors)
+            )
+        elif title:
+            queryset = queryset.filter(
+                title__icontains=title
+            )
+        return queryset
```

Comment:
The `return queryset` statement within the `elif title` block should be aligned with other return statements to prevent premature method exit when the title filter is applied.